pkg/reconciler/labeler/labelerv2_test.go

```diff
@@ -424,6 +441,28 @@ func patchAddLabel(namespace, name, key, value string) clientgotesting.PatchActi
 	return action
 }
 
+func patchAddRouteAndServingStateLabel(namespace, name, routeName string, now time.Time) clientgotesting.PatchActionImpl {
+	action := clientgotesting.PatchActionImpl{}
+	action.Name = name
+	action.Namespace = namespace
+
+	var state string
+	if routeName == "null" {
+		state = "Reserve"
+	} else {
+		state = "Active"
+		routeName = fmt.Sprintf("%q", routeName)
+	}
```

Comment:
I think this is simpler (and sprintf seems too heavy for quoting).
```suggestion
	state := string(RoutingStateReserve)
	if routeName != "null" {
		state = string(RoutingStateActive)
		routeName =`"`+routeName+`"`
	}
```